tests/installation/zdup.pm

```diff
@@ -131,6 +133,12 @@ sub run() {
         elsif ($out =~ $zypper_dup_finish) {
             last;
         }
+        elsif ($out =~ $zypper_retrieving or $out =~ $zypper_check_conflicts) {
+            # probably to avoid hitting black screen on video
+            send_key 'shift', 1;
+            # continue but do not drop zypper_dup_fileconflict check
```

Comment:
nah, the file conflicts once happened must after zypper_check_fileconflicts, and in this block it should continue the loop avoiding it goes to https://github.com/nilxam/os-autoinst-distri-opensuse/blob/more_check_zdup/tests/installation/zdup.pm#L155 , because in Line155 it will drop file conflicts check from $out. or you're confused $zypper_check_conflicts and $zypper_dup_conflicts?